lua/entities/gmod_tardis_interior/modules/sh_power.lua

```diff
@@ -10,6 +10,7 @@ if SERVER then
         self:SetPower(on)
     end
     function ENT:SetPower(on)
+        if not (self:CallHook("CanTogglePower")==true or self.exterior:CallHook("CanTogglePower")==true) then return end
```

Comment:
This hook should only exist on the exterior, convert anything using the interior one to call exterior

Reply:
Other way around? Module exists in the interior

Reply:
True, but maybe the module should exist on the exterior as it affects exterior functions such as flight, it's not an interior only thing